rust/noosphere-gateway/src/handlers/v0alpha2/mod.rs

```diff
@@ -0,0 +1,3 @@
+mod push;
+
+pub use push::*;
```

Comment:
Similar to the API in `noosphere-core`, I'd expect re-exports, and import `handlers::v0alpha2` within `noosphere-gateway`

Reply:
See comment above. The user-visible manifestation of these routes will be that there are routes at `v0alpha1/*` and `v0alpha2/*`, which is being done intentionally for migration/compatibility. It seems fitting that our code organization mirrors this ground truth.